pkg/model/deployment_chain.proto

```diff
@@ -0,0 +1,48 @@
+// Copyright 2021 The PipeCD Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package pipe.model;
+option go_package = "github.com/pipe-cd/pipe/pkg/model";
+
+import "validate/validate.proto";
+import "pkg/model/application.proto";
+import "pkg/model/common.proto";
+
+message DeploymentChain {
+    // The generated unique identifier.
+    string id = 1 [(validate.rules).string.min_len = 1];
+    // The ID of the project this environment belongs to.
+    string project_id = 2 [(validate.rules).string.min_len = 1];
+    // Deployment node contains all configuration required to perform deployment(s)
+    repeated DeploymentNode nodes = 3;
```

Comment:
Do this contain the first "node" ("app", "deployment")?
It would be nice if the comment could say that to be more understandable.

Reply:
It will contain the first "node" 👍 Will add note for that. We will use this nodes list to build for instance the UI graph on the deployment chain detail page.